core-backend/common/src/lib.rs

```diff
@@ -102,6 +102,7 @@ pub enum TrapExplanation {
 pub struct ExtInfo {
     pub gas_amount: GasAmount,
     pub allocations: BTreeSet<WasmPageNumber>,
```

Comment:
can't we pre-store initial allocations and keep here only actual after execution?

Reply:
ok